### PR TITLE
Licensing updates

### PR DESCRIPTION
**Pull request by JohanBjerning**

### CODE REVIEW

**Comment by qlikossbuild** [comment]
Deploy preview for *core-website* ready!

Built with commit 3f40e5b4dc464e1acaf0e655177bf041b79792c6

https://deploy-preview-522--core-website.netlify.com

File: docs/licensing.md [hunk 1]
```diff
@@ -1,49 +1,187 @@
-# Licensing
+<style>
```
Comment (peol) [inline docs/licensing.md]: Move this to a CSS file somewhere :)

File: mkdocs.yml [hunk 2]
```diff
@@ -61,7 +64,7 @@ nav:
     - December 2018: updates/2018-12.md
     - November 2018: updates/2018-11.md
     - October 2018: updates/2018-10.md
-  - Licensing: licensing.md
+  - Pricing: licensing.md
```
Comment (peol) [inline mkdocs.yml]: Wouldn't this make the URL 'licensing'?

Comment (hrigner) [reply mkdocs.yml]: Yes.. but there are still info about the licensing model on the page :)

Comment (JohanBjerning) [reply mkdocs.yml]: Yes but I didn't get the redirect to work as I wanted to I will take that in a separate PR

File: docs/stylesheets/pricing.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+}
+.md-content__icon {
+    display: none;
+}
+.cards-pricing {
+    border: 2px;
+    border-style: solid;
+    margin-top: 10px;
+    margin-right: 10px;
+    float: left;
```
Comment (sublibra) [inline docs/stylesheets/pricing.css]: I guess we don't mind alphabetical order in css files. :-)

**Pull request merged**